front/src/generated/graphql.tsx

```diff
@@ -22,6 +22,12 @@ export type AffectedRows = {
   count: Scalars['Int'];
 };
 
+export type Analytics = {
```

Comment:
I'm surprised about these changes (Analytics, + Events). Is it because we forgot to refresh the schema in a previous PR? We might want to add a CI check about that

Reply:
Yep, doesn't seem related to this PR